src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -114,7 +114,7 @@ class CommandRequestImpl : public CommandImpl,
    * has exceed it's limit
    *
    */
-  virtual void onTimeOut();
+  virtual void onTimeOut(const std::string &info = "");
```

Comment:
@pold500 virtual function with default parameters is very very bad. It even pointed out in google coding style